composer.json

```diff
@@ -42,30 +42,32 @@
         "php-mock/php-mock-phpunit": "^2.0",
         "phpdocumentor/reflection-docblock": "^3.0 || ^4.0",
         "phpdocumentor/type-resolver": "^0.2.1 || ^0.3 || 0.4",
-        "phpunit/phpunit": "^6.1",
+        "phpunit/phpunit": "^6.1 || ^7.0",
```

Comment:
why do we need to change phpunit version?

Reply:
I was wondering why we were not using the latest already actually. I can revert @soyuka

Reply:
Allowing both versions is better, keep it as is.

Reply:
can't this be why appveyor is not behaving correctly anymore?